env.example

```diff
@@ -338,3 +341,40 @@ RESTART_POLICY=unless-stopped
 
 # Authenticate using external service or just focus external auth window if there is one already.
 # TOKEN_AUTH_URL=https://auth.meet.example.com/{room}
+
+# Jicofo config
+JICOFO_SHORT_ID=1
+
+# The bridge selection strategy. The built-in strategies are:
+# --> SingleBridgeSelectionStrategy: Use the least loaded bridge, do not split a conference between bridges (Octo).
+# --> SplitBridgeSelectionStrategy: Use a separate bridge for each participant (for testing).
+# --> RegionBasedBridgeSelectionStrategy: Attempt to put each participant in a bridge in their local region (i.e. use Octo for geo-location).
+# --> IntraRegionBridgeSelectionStrategy: Use additional bridges when a bridge becomes overloaded (i.e. use Octo for load balancing). 
+OCTO_BRIDGE_SELECTION_STRATEGY=SingleBridgeSelectionStrategy
+
+# Configure the codecs and RTP extensions to be used in the offer sent to clients.
+ENABLE_CODEC_VP8=true
+ENABLE_CODEC_VP9=true
+ENABLE_CODEC_H264=true
+
+# The maximum number of participants in a single conference to put on one bridge (use -1 for no maximum).
+#MAX_BRIDGE_PARTICIPANTS=-1
+          
+# Whether jicofo should perform periodic health checks to the connected bridges.
+JICOFO_ENABLE_BRIDGE_HEALTH_CHECKS=true
+
+# Whether to automatically grant the 'owner' role to the first participant in the conference (and subsequently to 
+# the next in line when the current owner leaves).
+#ENABLE_AUTO_OWNER=true
+
+# How long to wait for the initial participant in a conference.
+#JICOFO_CONF_INITIAL_PARTICIPANT_WAIT_TIMEOUT="15 seconds"
+
+# How long a participant's media session will be kept alive once it remains the only participant in the room.
+#JICOFO_CONF_SINGLE_PARTICIPANT_TIMEOUT="20 seconds"
+            
+# jibri-sip  
+#ENABLE_JIBRI_SIP=0
```

Comment:
Is this intentionally not implemented?

Reply:
Yes. I have no idea about Jibri-sip(never used it). I think @saghul can help here.